server/model/MatchModel.js

```diff
@@ -0,0 +1,15 @@
+export class MatchModel {
+  constructor(userNameArr, userIDArr) {
+
+    // variables for storing who starts the match and their unique IDs
+    this.userNameArr = userNameArr;   // array datatype
+    this.userIDArr = userIDArr;       // array datatype
```

Comment:
```suggestion
    this.userIds = userIDArr;       // array datatype
```

Instead of naming it an "array", `Arr`, use validation before the data enters the db instead of the attribute determining what type of data it should be

Reply:
Changed the `userIDArr` to `userIDs`